.github/workflows/dojo-e2e.yml

```
@@ -0,0 +1,114 @@
+          COPILOT_CLOUD_STAGING_PUBLIC_API_KEY: ${{ secrets.COPILOT_CLOUD_STAGING_PUBLIC_API_KEY }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          GITHUB_ACTIONS_RUN_ID: ${{ github.run_id }}
+
+      - name: Comment post-deployment
+        if: always()
+        uses: thollander/actions-comment-pull-request@v3
+        with:
+          pr-number: ${{ inputs.pr_number }}
+          comment-tag: test-run-status-update
+          file-path: ./examples/e2e/test-results/test-run-comment.md
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)